lib/xeroizer/models/credit_note.rb

```diff
@@ -57,6 +57,7 @@ class CreditNote < Base
       decimal       :total, :calculated => true
       datetime_utc  :updated_date_utc, :api_name => 'UpdatedDateUTC'
       string        :currency_code
+      decimal       :currency_rate, :calculated => true
```

Comment:
According to the docs, this can either be specified by the end user or it gets calculated on the fly if omitted. #200 is probably a better version of this.